common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -423,6 +426,8 @@ object DotcomRenderingDataModel {
 
     val matchData = makeMatchData(page)
 
+    val selectedTopics = topicResult.map(x => Seq(Topic(x.`type`, x.name)))
+
```

Comment:
If you’re not giving it a meaningful name, you can always use the underscore!

```suggestion
    val selectedTopics = topicResult.map(_ => Seq(Topic(_.`type`, _.name)))

```

Reply:
```suggestion
    val selectedTopics = topicResult.map(Seq(Topic(_.`type`, _.name)))
```

even